modules/CircularLayout/src/main/java/org/gephi/layout/plugin/circularlayout/circlelayout/CircleLayout.java

```diff
@@ -0,0 +1,281 @@
+                           NbBundle.getMessage(getClass(), "CircleLayout.BoolFixedDiameter.desc"),
+                           "isBoolFixedDiameter", "setBoolFixedDiameter"));
+         properties.add(LayoutProperty.createProperty(
+                           this, Double.class,
+                           NbBundle.getMessage(getClass(), "CircleLayout.Diameter.name"),
+                           PROPERTIES_CATEGORY,
+                           NbBundle.getMessage(getClass(), "CircleLayout.Diameter.desc"),
+                           "getDiameter", "setDiameter"));
+         properties.add(LayoutProperty.createProperty(
+                           this, String.class,
+                           NbBundle.getMessage(getClass(), "CircleLayout.NodePlacement.NodeOrdering.name"),
+                           PROPERTIES_CATEGORY,
+                           NbBundle.getMessage(getClass(), "CircleLayout.NodePlacement.NodeOrdering.desc"),
+                           "getNodePlacement", "setNodePlacement", LayoutComboBoxEditor.class));
+         properties.add(LayoutProperty.createProperty(
+                           this, CircularDirection.class,
+                           NbBundle.getMessage(getClass(), "CircleLayout.NodePlacement.Direction.name"),
+                           PLACEMENT_CATEGORY,
+                           NbBundle.getMessage(getClass(), "CircleLayout.NodePlacement.Direction.desc"),
+                           "getNodePlacementDirection", "setNodePlacementDirection", RotationComboBoxEditor.class));
+         properties.add(LayoutProperty.createProperty(
+                           this, Boolean.class,
+                           NbBundle.getMessage(getClass(), "CircleLayout.NodePlacement.NoOverlap.name"),
+                           PLACEMENT_CATEGORY,
+                           NbBundle.getMessage(getClass(), "CircleLayout.NodePlacement.NoOverlap.desc"),
+                           "isNodePlacementNoOverlap", "setNodePlacementNoOverlap"));
+         properties.add(LayoutProperty.createProperty(
+                           this, Boolean.class,
+                           NbBundle.getMessage(getClass(), "CircleLayout.Transition.EnableTransition.name"),
+                           TRANSITION_CATEGORY,
+                           NbBundle.getMessage(getClass(), "CircleLayout.Transition.EnableTransition.desc"),
+                           "isNodePlacementTransition", "setNodePlacementTransition"));
+         properties.add(LayoutProperty.createProperty(
+                           this, Double.class,
+                           NbBundle.getMessage(getClass(), "CircleLayout.Transition.TransitionSteps.name"),
+                           TRANSITION_CATEGORY,
+                           NbBundle.getMessage(getClass(), "CircleLayout.Transition.TransitionSteps.desc"),
+                           "getTransitionSteps", "setTransitionSteps"));
+      }
+      catch (Exception e) {
+         e.printStackTrace();
```

Comment:
I would recommend to replace that with `Logger.getLogger(CircleLayout.class.getName()).log(Level.SEVERE, null, e)` so the error is propagated and will show up in Gephi's console. In general, all `System.out` and exception throwing should use this logger.